redisinsight/api/src/modules/cli/services/cli-analytics/cli-analytics.service.ts

```diff
@@ -108,38 +144,32 @@ export class CliAnalyticsService extends TelemetryBaseService {
     }
   }
 
-  sendCliCommandErrorEvent(
-    instanceId: string,
-    error: ReplyError,
-    additionalData: object = {},
-  ): void {
-    try {
-      this.sendEvent(
-        TelemetryEvents.CliCommandErrorReceived,
-        {
-          databaseId: instanceId,
-          error: error?.name,
-          command: error?.command?.name,
-          ...additionalData,
-        },
-      );
-    } catch (e) {
-      // continue regardless of error
-    }
-  }
-
-  sendCliConnectionErrorEvent(
+  sendConnectionErrorEvent(
     instanceId: string,
+    namespace: string,
     exception: HttpException,
     additionalData: object = {},
   ): void {
     this.sendFailedEvent(
-      TelemetryEvents.CliClientConnectionError,
+      this.getNamespaceEvent(TelemetryEvents.ClientConnectionError, namespace),
       exception,
       {
         databaseId: instanceId,
         ...additionalData,
       },
     );
   }
+
+  private getNamespaceEvent(event: TelemetryEvents, namespace: string = AppTool.CLI): string {
+    switch (namespace.toLowerCase()) {
+      case 'workbench': {
+        return `WORKBENCH_${event}`;
+      }
+      case 'cli': {
+        return `CLI_${event}`;
+      }
+      default:
+        return `CLI_${event}`;
+    }
```

Comment:
there is no need to duplicate `CLI_` prefix in the 'cli' and 'default' cases. Use default only instead

Reply:
done